### ISSUE TITLE
[java] UnnecessaryFullyQualifiedName doesn't flag same package FQCNs

### ISSUE THREAD

**Issue by jsotuyod** [pmd/pmd#1199]
**Affects PMD Version:** 6.4.0

**Rule:**
[`category/java/codestyle.xml/UnnecessaryFullyQualifiedName`](https://pmd.github.io/pmd-6.4.0/pmd_rules_java_codestyle.html#unnecessaryfullyqualifiedname)

**Description:**
Unnecessary FQCN of same package classes are not flagged.

**Code Sample demonstrating the issue:**

```
package org.foo;

public class SamePackage {
    public void convert(org.foo.Bar s) { // violation
    }
}
```

**Running PMD through:** Any
